spec/granite/form/model/validations/nested_spec.rb

```diff
@@ -143,6 +143,106 @@
       it do
         expect { subject.valid? }.not_to raise_error
         expect(subject).not_to be_valid
+        expect(subject.errors.count).to eq(1)
+      end
+    end
+  end
+
+  context 'object field is invalid and has representation of that field' do
+    before do
+      stub_class(:validated_object) do
+        include Granite::Form::Model::Validations
+        include Granite::Form::Model::Attributes
+        include Granite::Form::Model::Primary
+
+        primary :id
+        attribute :title, String
+        validates_presence_of :title
+      end
+
+      Main.class_eval do
+        include Granite::Form::Model::Associations
```

Comment:
I don't think you need associations

Reply:
Fair, doesn't needed for original `object field is invalid and referenced object does not include AutosaveAssociation` context also. Should I remove it?

Reply:
Yes